### PR TITLE
Update issue templates

### LINKED ISSUE #770: Add issue templates to this repo
- Proposal
  - What to change
  - Reasons for
  - Reasons against
  - Community support/impact
- Problem
  - What's wrong
  - What are some ideas for how to address it?

https://help.github.com/en/articles/creating-issue-templates-for-your-repository

### PR DESCRIPTION
**Pull request by Ivan-Brigidano**
Closes the already closed #770 
This adds a issue template to the repo.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/typo-report.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+name: Typo report
+about: Create a report to help us improve
+title: ''
+labels: typo
+assignees: ''
+
+---
+
+**Describe the bug**
```
Comment (hanwu85) [inline .github/ISSUE_TEMPLATE/typo-report.md]: Is it strange to use the word "bug" here?

Comment (Ivan-Brigidano) [reply .github/ISSUE_TEMPLATE/typo-report.md]:
```suggestion
**Describe the typo**
```

**Comment by hanwu85** [comment]
Why is travis still checking it?  Is it normal.  Seems taking too much time to me.

**Review by hanwu85 — APPROVED** [review]
Thanks!

**Pull request merged**